utils/json_profiles_merger.py

```diff
@@ -42,6 +43,9 @@ def main(argv):
   # Discard the first argument (the binary).
   input_profiles = argv[1:]
 
+  if not input_profiles:
+      raise RuntimeError("At least one profile must be provided!")
```

Comment:
This should be `ValueError` instead, to be consistent with the rest of the repository. Also please use 2 spaces for indentation.

Reply:
Good point, fixed :+1: